server/sdk/types/IVoipService.ts

```diff
@@ -15,5 +17,6 @@ export interface IVoipService {
 	getQueuedCallsForThisExtension(requestParams: any): Promise<IVoipConnectorResult>;
 	getExtensionList(): Promise<IVoipConnectorResult>;
 	getExtensionDetails(requestParams: any): Promise<IVoipConnectorResult>;
+	handleEvent(event: string, room: IRoom, user: IUser, comment?: string): Promise<void>;
```

Comment:
```suggestion
	handleEvent(event: 'voip-call-started', room: IRoom, user: IUser, comment?: string): Promise<void>;
```
You can also create a custom type for the possible values of event;

Reply:
Fixed.